datalad/distribution/create_sibling.py

```diff
@@ -124,9 +124,9 @@ def put(self, source, destination, recursive=False,
                 # destination directory already exists. With Python 3.8, we can
                 # make copytree() do the same with dirs_exist_ok=True. But for
                 # now, just rely on `cp`.
-                cmd = ["cp", "--recursive"]
+                cmd = ["cp", "-R"]
                 if preserve_attrs:
-                    cmd.append("--preserve")
+                    cmd.append("-p")
```

Comment:
eh, according to coverage we are not even testing this flag "directly" (well - git-annex would use it if it saw it available at "compile time") -- could you please tune up or extend some test for create_sibling so it tests with `preserve_attrs=True`?

Reply:
There does not appear to be any way to tell `create_sibling` to use `preserve_attrs` here.  It appears that no code that calls this method ever sets `preserve_attrs=True`.

Reply:
oh, indeed.  Could you please craft a dedicated to `_RunnerAdapter` unittest and test `put` with `preserve_attrs`?  then we would be sure it would work whenever we need to use it.